scss/_navbar.scss

```diff
@@ -456,7 +456,8 @@
         }
         // End mod
 
-        .navbar-nav-scroll {
+        .navbar-nav-scroll,
+        &.navbar-nav-scroll {
```

Comment:
Since we're not sure yet about a generic solution, isn't it safer to use `.overflow-visible` in `site/layouts/partials/docs-navbar.html`, don't modify this rule, and don't add a new explanation in `site/content/docs/5.3/components/orange-navbar.md` that might not work in all cases?